src/packit_deploy/__about__.py

```diff
@@ -1,4 +1,4 @@
 # SPDX-FileCopyrightText: 2023-present Alex Hill <alex.hill@gmail.com>
 #
 # SPDX-License-Identifier: MIT
-__version__ = "0.0.0"
+__version__ = "0.0.5"
```

Comment:
this went up to 0.0.5 because I was experimenting with pushing earlier versions to pypi!